apps/wallet/src/ui/app/pages/home/transfer-coin/TransferCoinForm/StepTwo.tsx

```diff
@@ -1,144 +1,61 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import BigNumber from 'bignumber.js';
-import cl from 'classnames';
-import { Field, Form, useFormikContext } from 'formik';
-import { useEffect, useRef, memo, useMemo } from 'react';
+import { useMemo } from 'react';
 
-import { Content, Menu } from '_app/shared/bottom-menu-layout';
-import Button from '_app/shared/button';
-import AddressInput from '_components/address-input';
-import Alert from '_components/alert';
-import Icon, { SuiIcons } from '_components/icon';
-import LoadingIndicator from '_components/loading/LoadingIndicator';
+import { Text } from '_app/shared/text';
+import { IconTooltip } from '_app/shared/tooltip';
+import { TxnAddress } from '_components/receipt-card/TxnAddress';
+import { TxnAmount } from '_components/receipt-card/TxnAmount';
 import { parseAmount } from '_helpers';
-import { useCoinDecimals, useFormatCoin } from '_hooks';
+import { useAppSelector, useFormatCoin, useCoinDecimals } from '_hooks';
 import { GAS_SYMBOL, GAS_TYPE_ARG } from '_redux/slices/sui-objects/Coin';
 
-import type { FormValues } from '../';
-
-import st from './TransferCoinForm.module.scss';
-
 export type TransferCoinFormProps = {
-    submitError: string | null;
-    coinSymbol: string;
     coinType: string;
-    gasBudgetEstimation: number | null;
     gasCostEstimation: number | null;
-    gasEstimationLoading?: boolean;
     onClearSubmitError: () => void;
```

Comment:
seems we don't use this anymore

Reply:
In the other PR https://github.com/MystenLabs/sui/pull/8353